src/main/kotlin/step2/StringCalculator.kt

```diff
@@ -0,0 +1,74 @@
+package step2
+
+class StringCalculator(
+    input: String,
+) {
+    private val operands: MutableList<Int> = mutableListOf()
+    private val operators: MutableList<ArithmeticOperator> = mutableListOf()
```

Comment:
```suggestion
    private val operands = mutableListOf<Int>()
    private val operators = mutableListOf<ArithmeticOperator>()
```

By using Kotlin's **type inference**, you can write code that is more concise and readable.
I think writing concise code aligns with writing more Kotlin-like code, making it idiomatic and expressive.